pkg/job_controller/pod.go

```diff
@@ -318,6 +318,11 @@ func (jc *JobController) ReconcilePods(
 	return nil
 }
 
+// CreatePod creates the pod
+func (jc *JobController) CreatePod(job interface{}, pod *v1.Pod) error {
```

Comment:
is it necessary to abstract a new function since argument `job` is not referenced ?